fetch.bs

```diff
@@ -5460,22 +5467,47 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
        <a for=request>body</a> cannot be recreated and that is why the buffer is needed.
       </div>
 
-     <li><p>Set <var>timingInfo</var>'s
-     <a for="fetch timing info">final network-response start time</a> to the
-     <a for=/>coarsened shared current time</a> given <var>fetchParams</var>'s
-     <a for="fetch params">cross-origin isolated capability</a>, immediately after the user agent's
-     HTTP parser receives the first byte of the response (e.g., frame header bytes for HTTP/2 or
-     response status line for HTTP/1.x).
+     <li>
+      <p>While true:
+      <ol>
+       <li><p>Wait until all the <a for=/>headers</a> are transmitted.
 
-     <li><p>Wait until all the <a for=/>headers</a> are transmitted.
+       <li><p>If <var>timingInfo</var>'s
+       <a for="fetch timing info">final network-response start time</a> is 0, set
+       <var>timingInfo</var>'s <a for="fetch timing info">final network-response start time</a> to
+       <a for=/>coarsened shared current time</a> given <var>fetchParams</var>'s
+       <a for="fetch params">cross-origin isolated capability</a>, immediately after the user
+       agent's HTTP parser receives the first byte of the response (e.g., frame header bytes for
+       HTTP/2 or response status line for HTTP/1.x).
 
-     <li>
-      <p>Any <a for=/>responses</a> whose <a for=response>status</a> is in the range 100 to 199,
-      inclusive, and is not 101, are to be ignored, except for the purposes of setting
-      <var>timingInfo</var>'s <a for="fetch timing info">final network-response start time</a> above.
+       <li><p>Let <var>status</var> be <a for=response>status</a> of <a for=/>response</a>.
+
+       <li><p>If <var>status</var> is in the range 100 to 199, inclusive:
+        <ol>
+         <li><p>If <var>status</var> is 101, <a for=iteration>break</a>.
+
+         <li><p>If <var>status</var> is 103 and <var>fetchParams</var>'s
+         <a for="fetch params">process early hints response</a> is non-null, then:
+          <ol>
+           <li><p><a for=/>Assert</a>: <var>request</var>'s <a for=request>mode</a> is
+           "<code>navigate</code>".
 
-      <p class="note no-backref">These kind of <a for=/>responses</a> are eventually followed by a
-      "final" <a for=/>response</a>.
+           <p class=note>The following step might trigger other fetches. The assertion
```

Comment:
Triggering other fetches is not a problem, the problem is that we make some notification before checking CORS. In this case it is fine because the request is not CORS eligible.